### PR TITLE
feat: Initial integration of aero workflow

### PR DESCRIPTION
**Pull request by prmukherj**
Initial integration of aero-workflow to PyFluent.

One can connect to a running solver in aero mode or can launch a fluent session in aero mode.

>>> pyfluent.launch_fluent(mode="solver_aero")

### CODE REVIEW

File: src/ansys/fluent/core/session_solver_aero.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        # TODO: Update Aero DM
+        self.scheme_eval.scheme_eval(
+            f"""(gui-aero-project-add-workflow-cb #f "{case_file_name}" #f #f)"""
+        )
+
+    def open_simulation(self, simulation_file_name: str):
+        """Open a saved simulation."""
+        # TODO: Update Aero DM
+        self.scheme_eval.scheme_eval(
+            f"""(aero-open-simname "{simulation_file_name}")"""
```
Comment (dswitche) [inline src/ansys/fluent/core/session_solver_aero.py:79]: @prmukherj can we replace the shorthand function here with the primary function? 

aero-open-simname -> aero-server-project-open-simulation

File: src/ansys/fluent/core/session_solver_aero.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+    def new_project(self, project_name: str):
+        """Define a new project."""
+        # TODO: Update Aero DM
+        self.scheme_eval.scheme_eval(f"""(prjapp-new-project-cb #f "{project_name}")""")
+
+    def open_project(self, project_name: str):
+        """Open a saved project."""
+        # TODO: Update Aero DM
+        self.scheme_eval.scheme_eval(f"""(aero-open-prjname "{project_name}")""")
```
Comment (dswitche) [inline src/ansys/fluent/core/session_solver_aero.py:66]: @prmukherj can we replace the shorthand function name here with the primary function? 

aero-open-prjname -> prjapp-project-open-project-cb
which has 2 arguments. 1st is hardcoded as #f, and 2nd is project_name

**Review by dswitche — APPROVED** [review]
looks good, I made 2 small comments. thanks for adding this prithwish!

File: src/ansys/fluent/core/session_solver_aero.py (hunk 1 above)
Comment (prmukherj) [reply src/ansys/fluent/core/session_solver_aero.py:79]: sure, we can. I'll do it. Thanks.

Comment (prmukherj) [reply src/ansys/fluent/core/session_solver_aero.py]: Done, Thank you.

File: src/ansys/fluent/core/session_solver_aero.py (hunk 2 above)
Comment (prmukherj) [reply src/ansys/fluent/core/session_solver_aero.py]: Done, Thank you.

File: src/ansys/fluent/core/session_solver_aero.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
+"""Module containing class encapsulating Fluent connection.
+
+**********PRESENTLY SAME AS SOLVER WITH A SWITCH TO SOLVER***********
+"""
```
Comment (mkundu1) [inline src/ansys/fluent/core/session_solver_aero.py:4]: Please update this comment

File: src/ansys/fluent/core/session_solver_aero.py [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+    def open_simulation(self, simulation_file_name: str):
+        """Open a saved simulation."""
+        # TODO: Update Aero DM
+        self.scheme_eval.scheme_eval(
+            f"""(aero-server-project-open-simulation "{simulation_file_name}")"""
+        )
+
+    @property
+    def _flserver(self):
```
Comment (mkundu1) [inline src/ansys/fluent/core/session_solver_aero.py:85]: This can be an instance-level attribute.

Comment (mkundu1) [reply src/ansys/fluent/core/session_solver_aero.py:85]: As discussed, we shall address this later.

**Review by mkundu1 — APPROVED** [review]

**Review by hpohekar — APPROVED** [review]

**Pull request merged**